packages/fiori-freestyle-writer/test/common.ts

```diff
@@ -9,6 +10,9 @@ export const testOutputDir = join(__dirname, '/test-output');
 
 export const debug = prepareDebug();
 
+/**
```

Comment:
Empty jsdocs?

Reply:
fixed in https://github.com/SAP/open-ux-tools/pull/689/commits/028c6ad39d339a95190a55c617b6f19ebf72348b